plugins/toolbox/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@drodil/backstage-plugin-toolbox",
-  "version": "1.9.2",
+  "version": "1.10.0",
```

Comment:
No need to update the version, CI will take care of that once the release is ready for shipping 🛥️